packages/__docs__/src/Figure/index.js

```diff
@@ -44,8 +44,22 @@ import generateComponentTheme from './theme'
 
 import { Heading } from '../Heading'
 
-class FigureItem extends ComponentIdentifier {
+class FigureItem extends Component {
```

Comment:
I refactored this, because ComponentIdentifier was marked as deprecated, and was throwing warnings.

Reply:
yep, that was annoying